hc-sr04-range-sensor/Makefile

```diff
@@ -61,6 +61,14 @@ OUT := out
 ELF0 := $(OUT)/pru-halt.elf
 ELF1 := $(OUT)/hc-sr04-range-sensor.elf
 
+#Installation, start stop related variable
+PRU0        :=/sys/class/remoteproc/remoteproc1
```

Comment:
Please rename to RPROC_PRU0. Otherwise it could easily be confused with ELF0 output binary for PRU0.